src/matrixvariates.jl

```diff
@@ -226,11 +226,15 @@ rows and `size(d, 2)` columns, or an array of matrices of size `size(d)`.
 """
 loglikelihood(d::MatrixDistribution, X::AbstractMatrix{<:Real}) = logpdf(d, X)
 function loglikelihood(d::MatrixDistribution, X::AbstractArray{<:Real,3})
-    (size(X, 1), size(X, 2)) == size(d) || throw(DimensionMismatch("Inconsistent array dimensions."))
-    return sum(i -> _logpdf(d, view(X, :, :, i)), axes(X, 3))
+    (size(X, 1), size(X, 2)) == size(d) || throw(DimensionMismatch("inconsistent array dimensions"))
+    # we use pairwise summation (https://github.com/JuliaLang/julia/pull/31020)
```

Comment:
```suggestion
    # we use pairwise summation (https://github.com/JuliaLang/julia/pull/31020)
    # to compute sum(i -> _logpdf(d, view(X, :, :, i)), axes(X, 3))
```

Reply:
I added a similar comment, somehow I missed this suggestion.